README.md

```diff
@@ -1,2 +0,0 @@
-# Shorts
-Shorts app front end code. Shorts takes a user's locations and recommends whether or not to wear shorts
```

Comment:
Why are you deleting the readme? its good to have one

Reply:
Yeah I deleted it thinking that since I already had one in my Master branch I wouldn't need it here. I'll see if I can add it back